lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -13,6 +13,7 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
yes, this was because I added ToJSON/FromJSON instances of ByteString. Now, as everything is taking place inside `PostExternalTransactionData` instances they are not present anymore,